src/main/java/com/agilefaq/codedoctorsample/App.java

```diff
@@ -12,7 +16,220 @@ public static int calc2(int a1, int a2) {
         return a1 - a2;
     }
 
-    public static int calc3(int a1, int a2) {
+    public static int multiply(int a1, int a2) {
+        Point point = new Point();
+        Point point1 = new Point();
+        if (point1 == point) {
+            System.out.println("Equal");
+        }
         return a1 * a2;
     }
+
+    public static int multiply4(int a1, int a2) {
+
+        Connection conn = null;
+        try {
+            conn = DriverManager.getConnection("jdbc:mysql://localhost/test?" +
+                    "user=steve&password=blue");
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        String uname = "steve";
+        String password = "blue";
+        try {
+            conn = DriverManager.getConnection("jdbc:mysql://localhost/test?" +
+                    "user=" + uname + "&password=" + password);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "&password=" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "user=" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1192)